include/dynamatic/Dialect/Handshake/HandshakeOps.td

```diff
@@ -1103,6 +1121,8 @@ def BundleOp : Handshake_Op<"bundle", [Pure]> {
         validReadyType
       });
     }]>,
+    // A builder for bundling ChannelType
+    // Implemented in C++
```

Comment:
same here, I would write
"ChannelType Builder (can be used in c++)"

is the can be used in c++ necessary?

Reply:
I wanted to mean the content of this builder is implemented in the C++ file `HandshakeOps.cpp`, apart from the ControlType Bundler
I don't know why one is implemented here and another is not
I guess Lucas wanted to implement relatively large methods in separate files